package.json

```diff
@@ -2,7 +2,7 @@
   "name": "web",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://www.elixirbench.org",
+  "homepage": "https://elixir-bench.gigalixirapp.com",
```

Comment:
? Shouldn't this stay as is because that's where we'll have it deployed and that's where it should go?